### ISSUE TITLE
Add DOIs to bibtex output for @TECHREPORT

### ISSUE THREAD

**Issue by aaccomazzi** [adsabs/export_service#110]
The current export service does not include the DOI field for a TECHREPORT publication type.  We should add this field not only for this document type, but for all records which do have a DOI (classic already does this).

See e.g. https://ui.adsabs.harvard.edu/abs/2013ivoa.rept.1213D
and http://adsabs.harvard.edu/cgi-bin/nph-bib_query?bibcode=2013ivoa.rept.1213D&data_type=BIBTEX&db_key=AST&nocookieset=1

**Comment by golnazads** [comment]
fixed.
```
@TECHREPORT{2013ivoa.rept.1213D,
       author = {{Demleitner}, Markus and {Harrison}, Paul and {Taylor}, Mark},
        title = \"{TAP Implementation Notes Version 1.0}\",
      journal = {IVOA Note 13 December 2013},
    booktitle = {IVOA Note 13 December 201},
         year = \"2013\",
       editor = {{Demleitner}, Markus},
        month = \"Dec\",
        pages = {1213},
          doi = {10.5479/ADS/bib/2013ivoa.rept.1213D},
       adsurl = {https://ui.adsabs.harvard.edu/abs/2013ivoa.rept.1213D},
      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
```

**Comment by golnazads** [comment]
released to dev on 6/4 and Build #260
